packages/capabilities/src/types.ts

```diff
@@ -158,51 +161,118 @@ export type Space = InferInvokedCapability<typeof space>
 export type SpaceInfo = InferInvokedCapability<typeof info>
 
 // filecoin
-export type FILECOIN_PROCESSING_STATUS = 'pending' | 'done'
-export interface FilecoinAddSuccess {
-  piece: PieceLink
+export interface DataAggregationProof {
+  /**
+   * Proof the piece is included in the aggregate.
+   */
+  inclusion: InclusionProof
+  auxDataType: uint64
+  auxDataSource: SingletonMarketSource
+}
+export interface InclusionProof {
+  /**
+   * Proof of inclusion of the client's data segment in the data aggregator's
+   * Merkle tree (includes position information). i.e. a proof that the root
+   * node of the subtree containing all the nodes (leafs) of a data segment is
+   * contained in CommDA.
+   */
+  subtree: ProofData
+  /**
+   * Proof that an entry for the user's data is contained in the index of the
+   * aggregator's deal. i.e. a proof that the data segment index constructed
+   * from the root of the user's data segment subtree is contained in the index
+   * of the deal tree.
+   */
+  index: ProofData
 }
-export interface FilecoinAddFailure extends Ucanto.Failure {
-  name: string
+export interface SingletonMarketSource {
+  dealID: uint64
 }
 
-export interface AggregateAddSuccess {
+export interface FilecoinOfferSuccess {
+  /**
+   * Commitment proof for piece.
+   */
   piece: PieceLink
-  aggregate?: PieceLink
-}
-export interface AggregateAddFailure extends Ucanto.Failure {
-  name: string
 }
+export type FilecoinOfferFailure = Ucanto.Failure
```

Comment:
We are missing here a `ContentNotFoundError` per spec https://github.com/web3-storage/specs/blob/feat/update-filecoin-spec/w3-filecoin.md#filecoinoffer